docs/organizer-guide/meetups/starting.rst

```diff
@@ -14,176 +13,186 @@ It contains a lot of the knowledge that we have put here:
 Write the Docs Worldview
 ------------------------
 
-Our worldview is important for starting a meetup.
-We aren't just another tech writing meetup,
-but hope to be welcoming to the vast parts of the tech industry that care about documentation.
+Yes, Write the Docs has a worldview.
+We include everyone who is interested in excellent documentation: tech writers, developers, engineers, QA, product managers, support engineers and more.
```

Comment:
Yes, Write the Docs has a worldview. It's inclusive. We welcome everyone who is interested in creating great documentation that helps people develop, use, and maintain products and services. Our community includes technical writers, developers of all flavors, QA, product managers, support engineers, designers, librarians, scientists, and more. 



So the following text is something that I cobbled together from reading the WTD website back in February: 
We consider everyone who cares about communication, documentation, and their users to be a member of our community. This can be programmers, tech writers, customer support, marketers, and anyone else who wants people to have great experiences with software.